util/bump_version.sh

```diff
@@ -43,7 +43,7 @@ read_versions() {
 # in: $1
 read_proto_ver() {
 	local ref=$1
-	git show "$ref":src/network/networkprotocol.h | grep -oE 'LATEST_PROTOCOL_VERSION [0-9]+' | tr -dC 0-9
+	git show "$ref":src/network/networkprotocol.cpp | grep -oE 'LATEST_PROTOCOL_VERSION =\s*[0-9]+' | tr -dC 0-9
```

Comment:
well no, this needs to work for old revisions too :)

Reply:
Would you please be so nice to explain when exactly this is needed? This script is generally at the same revision as the rest of the branch (which is subject to the bump).

Reply:
it checks the protocol version of the previous tag